[docs] Correct info about updating nodes (#16629)
deckhouse/deckhouse: modules/040-node-manager/docs/FAQ.md

Refer to the description of the [NodeGroup](cr.html#nodegroup) custom resource for more information about the parameters.

> When the `InstanceClass` or `instancePrefix` parameters are modified, a process similar to updating a Deployment in Kubernetes will occur (in this case, changing `InstanceClass` results in creating a new `MachineSet`, and changing `instancePrefix` results in creating a new `MachineDeployment`). New instances will be created according to the value of the `cloudInstances.maxSurgePerZone` parameter, and as they are created, the old instances will be gradually removed with a drain operation performed beforehand.

Reviewer: ```suggestion
When the `InstanceClass` or `instancePrefix` parameters are modified, the process is similar to updating a Deployment in Kubernetes (changing `InstanceClass` results in creating a new MachineSet, and changing `instancePrefix` results in creating a new MachineDeployment). New instances will be created according to the value of the [`cloudInstances.maxSurgePerZone`](/modules/node-manager/cr.html#nodegroup-v1-spec-cloudinstances-maxsurgeperzone) parameter. As the new instances are created, the old ones will be gradually removed, with a drain operation performed beforehand.
```
